src/pydvl/value/shapley/montecarlo.py

```diff
@@ -234,10 +231,13 @@ def permutation_montecarlo_shapley(
         map_kwargs=dict(max_permutations=iterations_per_job, progress=progress),
         reduce_kwargs=dict(axis=0),
         config=config,
-        chunkify_inputs=False,
         n_jobs=n_jobs,
     )
-    full_results = map_reduce_job(u_id)[0]
+    if n_jobs == 1:
+        input_ = u
+    else:
+        input_ = tuple(repeat(u, times=n_jobs))
```

Comment:
Why the case handling? a tuple of `(u,)` should also be ok. As a matter of fact, I'd expect a map-reduce op to always take a sequence and never a single item.